deploy/operator.yaml

```diff
@@ -15,8 +15,7 @@ spec:
       serviceAccountName: heimdall
       containers:
         - name: heimdall
-          # Replace this with the built image name
-          image: REPLACE_IMAGE
+          image: quay.io/integreatly/heimdall-operator:master
```

Comment:
Went with the same pattern as other operators here where the image in this file is set to the latest released version. Will keep it as master until we are ready to do an initial release.